### PR TITLE
Add hell and less hell maps

### PR DESCRIPTION
**Pull request by joshhhhhhhhhh**

### CODE REVIEW

File: Hell & Less Hell/README.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#Description
```
Comment (Deflaktor) [inline Hell & Less Hell/README.txt:1]:
```suggestion
# Hell
```

**Comment by Deflaktor** [comment]
Thanks for the pull request. I have fixed the format for the readme and will playtest the maps that you submitted.

**Comment by Hubcapp** [comment]
this map is appropriately named! looking forward to trying it out.

**Comment by Hubcapp** [comment]
Screen Shot 2020-10-23 at 1 29 25

**Comment by joshhhhhhhhhh** [comment]
Lmaooooo please let me know what you guys think of it

**Comment by Deflaktor** [comment]
👍

**Comment by Berbasaur** [comment]
Hello fellow fortune street gamers. My name is Berbasaur and I am the person in the above picture who showed interest in the custom fortune street map known as 'Hell'. I have played the map with 3 other friends in a 90 turn game and won. I want to thank the creator of this map, because it was very well made. Jolly good show!

**Comment by Hubcapp** [comment]
We decided to play "Hell Map" as opposed to "Less Hell Map".

Despite its chaotic & bizarre aesthetic, **this map makes it to the top 5 out of all Fortune Street boards we have played** (And we have played pretty close to all of them). The multitude of islands starts to make sense after a while. As opposed to stock maps like "Yoshi's Island", where the strategy is basically to rush the island & try to conquer it first, "Hell Map" lets everyone be an Island Czar at the same time.  Everyone can get their own monopoly-trap island simultaneously, which results in more fun gameplay, since no one feels left out and unable to make a comeback.

The usually lower-tier Home is pretty devastating here, and it truly does feel like you are in hell. In my opinion, the islands earlier into the bank-loop are more powerful, because Island Czars will be able to prevent you from levelling up & buying stocks at all unless you roll very high after getting lucky enough to warp out. Cannons often send you backwards in the bank-loop, making it even harder to reach the bank.

**Possible changes:**

- Buff promotion amounts to reflect the fact that it is hard to level up.
- It might make sense to put a stock market half way through the bank loop, since I felt pretty stock-starved, but also it works really well right now to have stocks be harder than usual to get.

Implementing these might detract from the character of the map & the feeling of being in hell, but it would also make it feel less like you are in hell.

All in all, great map. Thanks so much for sharing. :-)

**Pull request merged**